packages/deployments/contracts/contracts/lib/Connext/AssetLogic.sol

```diff
@@ -31,7 +31,7 @@ library AssetLogic {
     if (_assetId == address(0)) {
       // When transferring native asset to the contract, always make sure that the
       // asset is properly wrapped
-      if (msg.value != _specifiedAmount) revert AssetLogic__transferAssetToContract_notAmount();
+      if (msg.value < _specifiedAmount) revert AssetLogic__transferAssetToContract_notAmount();
```

Comment:
is there a reason the equality shouldn't be asserted?

Reply:
That is for the callback fee. When calling xcall with ETH, callback fee will be included in `msg.value`. So `msg.value` will be greater than `amount`.

Reply:
Fixed in `handleIncomingAsset` function